project/src/main/java/com/google/sps/utilities/PrefilledInformationConstants.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.utilities;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public final class PrefilledInformationConstants {
```

Comment:
Let's add a comment on where do the labels come from.

Reply:
Done